packages/run-protocol/src/vaultFactory/liquidation.js

```diff
@@ -3,30 +3,11 @@
 
 import { E } from '@endo/eventual-send';
 import { AmountMath } from '@agoric/ertp';
-import {
-  ceilMultiplyBy,
-  makeRatio,
-  offerTo,
-} from '@agoric/zoe/src/contractSupport/index.js';
+import { makeRatio, offerTo } from '@agoric/zoe/src/contractSupport/index.js';
 import { makeTracer } from '../makeTracer.js';
 
 const trace = makeTracer('LIQ');
 
-/**
- * @param {Amount<'nat'>} proceeds
- * @param {Amount<'nat'>} debt - after incurring penalty
- * @param {Amount<'nat'>} penaltyPortion
- */
-const partitionProceeds = (proceeds, debt, penaltyPortion) => {
```

Comment:
now that the math is split between two contracts, not worth having this combo fn